fuse_core/src/async_sensor_model.cpp

```diff
@@ -36,47 +36,60 @@
 #include <fuse_core/callback_wrapper.h>
 #include <fuse_core/graph.h>
 #include <fuse_core/transaction.h>
-#include <ros/callback_queue.h>
-
-#include <boost/make_shared.hpp>
 
 #include <functional>
 #include <utility>
 #include <string>
 
+#include <rclcpp/contexts/default_context.hpp>
 
 namespace fuse_core
 {
 
 AsyncSensorModel::AsyncSensorModel(size_t thread_count) :
   name_("uninitialized"),
-  spinner_(thread_count, &callback_queue_)
+  executor_thread_count_(thread_count)
 {
 }
 
-void AsyncSensorModel::graphCallback(Graph::ConstSharedPtr graph)
-{
-  callback_queue_.addCallback(
-    boost::make_shared<CallbackWrapper<void>>(std::bind(&AsyncSensorModel::onGraphUpdate, this, std::move(graph))),
-    reinterpret_cast<uint64_t>(this));
-}
 
 void AsyncSensorModel::initialize(
   const std::string& name,
   TransactionCallback transaction_callback)
 {
   // Initialize internal state
   name_ = name;
-  node_handle_.setCallbackQueue(&callback_queue_);
-  private_node_handle_ = ros::NodeHandle("~/" + name_);
-  private_node_handle_.setCallbackQueue(&callback_queue_);
+  std::string node_namespace = "";
+
+  rclcpp::Context::SharedPtr ros_context = rclcpp::contexts::get_global_default_context();
```

Comment:
Same comment about TODO to pass in either context or node

Reply:
https://github.com/locusrobotics/fuse/pull/284/commits/7f463a9ba85b274a53f618159fac00ae32b29f05